src/clusterfuzz/_internal/cron/triage.py

```diff
@@ -461,7 +461,7 @@ def should_throttle(self, testcase):
       if count_per_job < job_bugs_filing_max:
         self._bug_filed_per_job_per_24hrs[testcase.job_type] = count_per_job + 1
         return False
-      logs.log(
+      logs.log_error(
```

Comment:
hmm actually, not convinced  these  should be errors though. should  we  make them WARN instead?